src/aegis_ai/features/cve/__init__.py

```diff
@@ -22,7 +22,7 @@ class SuggestImpact(Feature):
 
     async def exec(self, cve_id: CVEID, static_context: Any = None):
         prompt = AegisPrompt(
-            user_instruction="Your task is to meticulously examine the provided CVE JSON object and suggest an overall impact rating for the CVE as well as suggest a related CVSS3 and CVSS4 score/vector. Ignore any mention of specific impact/severity (or CVSS) and come to independent conclusion.",
+            user_instruction="Your task is to meticulously examine the provided CVE JSON object and suggest an overall impact rating for the CVE as well as suggest a related CVSS3 score/vector. Ignore any mention of specific impact/severity (or CVSS) and come to independent conclusion.",
```

Comment:
all the prompts need a bit more structure ... I also think mixing analysis for both 3 (actually 3.1) and 4 causes problems as well.